python/cucim/setup.py

```diff
@@ -39,6 +39,7 @@ def read(*names, **kwargs):
     url='https://developer.nvidia.com/multidimensional-image-processing',
     packages=find_packages('src'),
     package_dir={'cucim': 'src/cucim'},
+    package_data={"": ["*.pyi"]},  # distribute
```

Comment:
Would it make sense to (somewhat?) align this with the `MANIFEST.in`?

Reply:
not sure under what conditions it matters (e.g. python setup.py sdist seems to have the .npy files etc that are listed in the MANIFEST.in without listing them again here). I went ahead and added it for consistency, though.

Reply:
Yeah the rule of thumb I try to use is...

* If we want to ship them, but not install them, use `MANIFEST.in`
* If we want to ship and install them, use `package_data`

Reply:
Think the suggestion in comment ( https://github.com/rapidsai/cucim/pull/575#discussion_r1275479020 ) is revises this to the list we want to install. Though please let me know if we are still missing things